internal/bgp/frr/testdata/TestLoggingConfiguration.golden

```diff
@@ -0,0 +1,9 @@
+
+log file /etc/frr/frr.log informational
+log stdout informational
```

Comment:
shouldn't it be a warning here instead of informational? as you configured the log level to the warning in the session manager of this test?

Reply:
you are definitely right! it's getting the default value. fixing it